src/components/RangeSlider.react.js

```diff
@@ -58,14 +59,51 @@ export default class RangeSlider extends Component {
             tipProps = tooltip;
         }
 
+        const truncatedMarks = this.props.marks
+            ? pickBy(
+                  (k, mark) => mark >= this.props.min && mark <= this.props.max,
+                  this.props.marks
+              )
+            : this.props.marks;
```

Comment:
A matter of personal preference, but you can also make this more compact with
```
const truncatedMarks = this.props.marks && pickBy(...);
```

JS does not return a boolean for `&&`, `||`, etc. but rather the last evaluated value.